GitBasic/MainWindow.xaml

```diff
@@ -48,7 +51,7 @@
                 <ColumnDefinition Width="0.3*" />
             </Grid.ColumnDefinitions>
 
-            <controls:ConsoleControl Grid.Column="0" />
+            <controls:ConsoleControl WorkingDirectory="{Binding WorkingDirectory.Value}" Grid.Column="0" />
```

Comment:
Data binding the View Model WorkingDirectory property into the Console Control WorkingDirectory property. (Note: they don't have to be called the same thing. I just thought it made sense to keep the name the same.)